GW2EIEvtcParser/EIData/MathUtils/Point2D.cs

```diff
@@ -173,5 +223,12 @@ public static bool IsInBox(Point2D p, Point2D p0, Point2D p1)
             var maxBB = new Point2D(Math.Max(p0.X, p1.X), Math.Max(p0.Y, p1.Y));
             return p >= minBB && p <= maxBB;
         }
+
+        public static Point2D RotatePointAroundPoint(Point2D centralPoint, Point2D rotationPoint, double angle)
+        {
+            double x = (rotationPoint.X - centralPoint.X) * Math.Cos(angle) - (rotationPoint.Y - centralPoint.Y) * Math.Sin(angle) + centralPoint.X;
```

Comment:
You should store the X - X and Y - Y values to reuse them. Same for the cos and sin.